M/MDAL/build_tarballs.jl

```diff
@@ -0,0 +1,70 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MDAL"
+version = v"0.8.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/lutraconsulting/MDAL/archive/refs/tags/release-$version.tar.gz", "0051495aff910b05d04efa792925cce377920ee02a03a89fa45529fd96cd2953"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+cd $WORKSPACE/srcdir/MDAL-*
+
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/rename-findsqlite-library.patch
+
+#NetCDF is the most restrictive dependency as far as platform availability, so we'll use it where applicable but disable it otherwise
+netcdf_platforms=(aarch64-linux-gnu i686-w64-mingw32 x86_64-apple-darwin x86_64-linux-gnu x86_64-w64-mingw32)
```

Comment:
Instead of hardcoding the list, maybe check if there is any `${libdir}/libnetcdf*.${dlext}` file?